src/core/battle/tests.rs

```diff
@@ -1256,13 +1257,28 @@ fn throw_bomb_push() {
         ),
         (
             "weak",
-            [component_agent_dull(), component_strength(1)].to_vec(),
+            [
+                component_agent_dull(),
+                component_strength(1),
+                component_blocker(Weight::Normal),
+            ]
+            .to_vec(),
+        ),
+        (
+            "heavy",
+            [
+                component_agent_dull(),
+                component_strength(1),
+                component_blocker(Weight::Normal),
+            ]
+            .to_vec(),
         ),
         ("bomb_push", Vec::new()),
     ]);
     let scenario = scenario::default()
         .object(P0, "thrower", PosHex { q: 0, r: 0 })
-        .object(P1, "weak", PosHex { q: 0, r: 3 });
+        .object(P1, "weak", PosHex { q: 0, r: 3 })
+        .object(P1, "heavy", PosHex { q: 1, r: 2 });
```

Comment:
Sooo, I added a heavy object to the test, I'm not sure I've understood all the "Ids", but I guess it has to do with order of coordinates checking before applying events, It looks like it's deterministic so it's good for the tests :+1: